mlir/utils/performance/perfRunner.py

```diff
@@ -489,8 +489,17 @@ def benchmarkExternal(cls, commandLine, paths: Paths, arch, envs=dict()):
             outs, errs = p1.communicate()
         return config.tableEntry(nanoSeconds)
 
-def getGemmConfigurations(fileName):
+OUT_DATATYPE = re.compile(r"-out_datatype[=\s*](.*)")
+def getGemmConfigurations(fileName, rocmlir_gen_flags):
```

Comment:
I'm not sure I like parsing out of rocmlir-gen flags here. If someone's trying to override the output data type, than they should use that.